sdk/core/azure-core-http-okhttp/src/main/java/com/azure/core/http/okhttp/OkHttpAsyncHttpClient.java

```diff
@@ -32,9 +43,16 @@
  * HttpClient implementation for OkHttp.
  */
 class OkHttpAsyncHttpClient implements HttpClient {
-    final OkHttpClient httpClient;
-    //
+
     private static final Mono<okio.ByteString> EMPTY_BYTE_STRING_MONO = Mono.just(okio.ByteString.EMPTY);
+    /**
+     * This constant defines a size of Flux based request where buffering in memory becomes less performant
+     * than streaming (which involves thread hops). The value has been established experimentally using
+     * Storage benchmarks.
+     */
+    private static final long BUFFERED_FLUX_REQUEST_THRESHOLD = 100 * 1024L;
+
+    final OkHttpClient httpClient;
```

Comment:
Can this be private?

Reply:
I tried that when I saw this but I discovered 10 tests that rely on this which are not easy to refactor.
This PR doesn't change visibility of this, so I don't want to mess with proxy test coverage here.